### PR TITLE
#RET 1636 Create positioning filter file

### PR DESCRIPTION
**Pull request by ivonx**

### CODE REVIEW

File: src/main/resources/filter/web/positioning.filter [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Positioning filter file for recheck. Please do not delete, even if it is empty.
+
+# You can ignore specific attributes like so: 
+# matcher: retestid=div-b4f23, attribute: font.*
+
+# To ignore whole subtrees, use e.g. xpath:
+# matcher: xpath=HTML[1]/BODY[1]/DIV[1]/DIV[1]/DIV[1]
+
+# To ignore attributes globally, use
```
Comment (beatngu13) [inline src/main/resources/filter/web/positioning.filter]: No punctuation (`:` or `.`) at the end.

File: src/main/resources/filter/web/positioning.filter [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+# To ignore whole subtrees, use e.g. xpath:
+# matcher: xpath=HTML[1]/BODY[1]/DIV[1]/DIV[1]/DIV[1]
+
+# To ignore attributes globally, use 
+# attribute=class
+
+# More details and examples can be found here:
+# https://retest.github.io/docs/recheck/how-ignore-works-in-recheck/
+
+# ignore positioning attributes
```
Comment (beatngu13) [inline src/main/resources/filter/web/positioning.filter]: Starts lowercase, missing punctuation.

File: src/main/resources/filter/web/positioning.filter [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+attribute=text-align-last
+attribute=text-indent
+attribute=top
+attribute=vertical-align
+attribute=white-space
+attribute=word-spacing
+attribute=word-wrap
+attribute=z-index
+
+#HTML attributes
```
Comment (beatngu13) [inline src/main/resources/filter/web/positioning.filter]: Missing space between `#` and `HTML`, missing punctuation.

File: src/main/resources/filter/web/positioning.filter [hunk 4]
```diff
@@ -0,0 +1,94 @@
+# Positioning filter file for recheck. Please do not delete, even if it is empty.
```
Comment (beatngu13) [inline src/main/resources/filter/web/positioning.filter]: Why not delete?

File: src/main/resources/filter/web/positioning.filter [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+# You can ignore specific attributes like so: 
+# matcher: retestid=div-b4f23, attribute: font.*
+
+# To ignore whole subtrees, use e.g. xpath:
```
Comment (beatngu13) [inline src/main/resources/filter/web/positioning.filter]: It is "XPath".

File: src/main/resources/filter/web/positioning.filter [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+# matcher: xpath=HTML[1]/BODY[1]/DIV[1]/DIV[1]/DIV[1]
+
+# To ignore attributes globally, use 
+# attribute=class
+
+# More details and examples can be found here:
+# https://retest.github.io/docs/recheck/how-ignore-works-in-recheck/
```
Comment (beatngu13) [inline src/main/resources/filter/web/positioning.filter]: I would rather expect this at the beginning or end of this file.

**Review by beatngu13 — DISMISSED** [review]
Comments follow different styles (start lowercase/uppercase, with/without punctuation, …). Please pick a particular style and be consistent.

File: src/main/resources/filter/web/positioning.filter (hunk 1 above)
Comment (ivonx) [reply src/main/resources/filter/web/positioning.filter]: I will fix that, thanks. But i copied the comments from the recheck.ignore file and the punctiation, the space, the  case  there should be changed too. Should i do that?

Comment (roesslerj) [reply src/main/resources/filter/web/positioning.filter]: Yes, please fix that in the "Master"-recheck.ignore file. Since these files are "internal", I don't think the comments are sensible here anyway – please remove them here.

File: src/main/resources/filter/web/positioning.filter [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+# ignore positioning attributes
+
+attribute=align-content
+attribute=align-items
+attribute=align-self
+attribute=background-origin
+attribute=background-position
+attribute=background-repeat
```
Comment (roesslerj) [inline src/main/resources/filter/web/positioning.filter]: This attribute is not related to positioning of elements, right?

File: src/main/resources/filter/web/positioning.filter [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+attribute=background -size
```
Comment (roesslerj) [inline src/main/resources/filter/web/positioning.filter]: Also not...

File: src/main/resources/filter/web/positioning.filter [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+attribute=background-clip
```
Comment (roesslerj) [inline src/main/resources/filter/web/positioning.filter]: Also not...

File: src/main/resources/filter/web/positioning.filter [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+attribute=border-bottom-width
+attribute=border-image-outset
+attribute=border-image-repeat
```
Comment (roesslerj) [inline src/main/resources/filter/web/positioning.filter]: Also not...

File: src/main/resources/filter/web/positioning.filter [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+attribute=border-image-width
+attribute=border-left-width
+attribute=border-radius
```
Comment (roesslerj) [inline src/main/resources/filter/web/positioning.filter]: Also not...

File: src/main/resources/filter/web/positioning.filter [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+attribute=border-top-width
+attribute=bottom
+attribute=box-decoration-break
+attribute=box-sizing
+attribute=caption-side
+attribute=column-gap
+attribute=column-rule-width
+attribute=column-width
+attribute=flex-basis
+attribute=attribute=flex-grow
```
Comment (roesslerj) [inline src/main/resources/filter/web/positioning.filter]: Copy&Paste error and all flex-attributes also not related to position, right?

File: src/main/resources/filter/web/positioning.filter [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+attribute=flex-wrap
+attribute=flex-shrink
+attribute=float
+attribute=font-size
+attribute=float
+attribute=font-size
+attribute=font-size-adjust
```
Comment (roesslerj) [inline src/main/resources/filter/web/positioning.filter]: also not

File: src/main/resources/filter/web/positioning.filter [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+attribute=font-weight
+attribute=grid-row-gap
+attribute=grid-column-gap
+attribute=left
+attribute=letter-spacing
```
Comment (roesslerj) [inline src/main/resources/filter/web/positioning.filter]: also not

File: src/main/resources/filter/web/positioning.filter [hunk 15]
```diff
@@ -0,0 +1,94 @@
...
+attribute=right
+attribute=tab-size
+attribute=text-align
+attribute=text-align-last
+attribute=text-indent
+attribute=top
+attribute=vertical-align
+attribute=white-space
+attribute=word-spacing
+attribute=word-wrap
```
Comment (roesslerj) [inline src/main/resources/filter/web/positioning.filter]: word-.* also not

File: src/main/resources/filter/web/positioning.filter [hunk 16]
```diff
@@ -0,0 +1,94 @@
...
+attribute=overflow-x
+attribute=overflow-y
+attribute=offsetHeight
+attribute=offsetWidth
+attribute=padding-top
+attribute=padding-right
+attribute=padding-bottom
+attribute=padding-left
+attribute=position
+attribute=resize
```
Comment (roesslerj) [inline src/main/resources/filter/web/positioning.filter]: also not

**Review by roesslerj — CHANGES_REQUESTED** [review]

File: src/main/resources/filter/web/positioning.filter [hunk 17]
```diff
@@ -0,0 +1,20 @@
...
+attribute=margin-left
+attribute=object-position
+attribute=padding-top
+attribute=padding-right
+attribute=padding-bottom
+attribute=padding-left
+attribute=position
+attribute=right
+attribute=top
+attribute=z-index
```
Comment (roesslerj) [inline src/main/resources/filter/web/positioning.filter:21]: I would also add our retest-specific outline.

**Review by roesslerj — CHANGES_REQUESTED** [review]

**Review by roesslerj — APPROVED** [review]
